src/components/Header.tsx

```diff
@@ -4,7 +4,7 @@ import Login from "./Login";
 const Header = (): JSX.Element => {
   return (
     <div className="Header__block">
-      <h1 className="Header__title">Example Client</h1>
+      <h1 className="Header__title">Example Client {process.env.RPC_URL}</h1>
```

Comment:
I think this is just debugging code, but if we did want it it should be:
```suggestion
      <h1 className="Header__title">Example Client {process.env.REACT_APP_CHAIN_HOST}</h1>
```

Reply:
Whoops. Good catch. This shouldn't be there.